docs/UserGuide.md

```diff
@@ -62,6 +62,8 @@ Survin is a desktop application for surveyors to use to keep track of people the
 -   Extraneous parameters for commands that do not take in parameters (such as `list`, `exit` and `clear`) will be ignored.<br>
     e.g. if the command specifies `list 123`, it will be interpreted as `list`.
 
+-   A unique person is defined by their name, phone number, and email. You cannot have 2 people with the same name, phone number, and email.
```

Comment:
Small caveat, I think that rephrasing as
> A person is uniquely characterised by their name, phone number and email.

would be unequivocally clearer.

Reply:
Yea, I agree. Thanks for the suggestion.